test/xpu/extended/run_test_with_skip.py

```diff
@@ -93,6 +91,16 @@
     # CPU result is not golden reference
     "test_compare_cpu_nn_functional_group_norm_xpu_bfloat16",
     "test_compare_cpu_nn_functional_group_norm_xpu_float16",
+
+    # Not implemented operators
+    "nn_functional_interpolate_linear",
+    "nn_functional_interpolate_bilinear",
+    "nn_functional_interpolate_trilinear",
+
+    # XPU kernel uses accumulate datatype with high accuracy
+    # Not in CUDA test scope too
+    "test_compare_cpu_nn_functional_interpolate_bicubic_xpu_bfloat16",
+    "test_compare_cpu_nn_functional_interpolate_bicubic_xpu_float16",
```

Comment:
Did you try if we don't use acc type, we would be same as CPU?

Reply:
> Did you try if we don't use acc type, we would be same as CPU?

no, and my previous conclusion is wrong
I think accuracy lost in half-precision is due to Bicubic interpolate algorithm is complex, which includes huge calculation steps
here is the link for algorithm formula 
https://en.wikipedia.org/wiki/Bicubic_interpolation#Bicubic_convolution_algorithm
bicubic need to do twice below computation
![image](https://github.com/intel/torch-xpu-ops/assets/22196524/8b49a891-d7ea-4002-8a70-ef4eba26c4f3)
I suspect accuracy may reduce during this process.